### PR TITLE
feat: Add support for bigquery_config

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

**Review by zied-elouaer — DISMISSED** [review]
Everything looks good. However, I suggest adding a test case that includes the new config, the same way we do for `push_config` [here](https://github.com/mineiros-io/terraform-google-pubsub-subscription/blob/mariux/bigquery_config/test/unit-complete/main.tf#L56).

**Comment by mariux** [comment]
@zied-elouaer valid point.... test added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0]
+
+### Added
```
Comment (soerenmartius) [inline CHANGELOG.md:12]: should this not be documented as a breaking change?

**Review by zied-elouaer — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**